device-api/src/device.rs

```diff
@@ -447,6 +447,13 @@ impl CoreRange {
             CoreRange::Range((s, e)) => (*s..=*e).contains(idx),
         }
     }
+
```

Comment:
Can you add a brief comment on `CoreRange::Range` describing that the range is inclusive? 😅

Reply:
Reflect at [c87341f](https://github.com/furiosa-ai/device-api/pull/101/commits/c87341f0539551194ad58ee8c76fa10425f64540)